sparrow-blog-main/src/main/resources/application-local.properties

```diff
@@ -0,0 +1,8 @@
+#
+spring.shardingsphere.datasource.master.url=jdbc:mysql://localhost:3306/sparrow_blog?useUnicode=true&characterEncoding=utf8&autoReconnect=true&allowMultiQueries=true&useSSL=false&serverTimezone=GMT%2B8
+spring.shardingsphere.datasource.master.username=root
+spring.shardingsphere.datasource.master.password=root
```

Comment:
Need to set the password to environment value

Reply:
> 

this is local config